packages/cli/src/rpc.ts

```diff
@@ -88,13 +89,7 @@ export async function runRpc(anvilOptions: AnvilOptions, rpcOptions: RpcOptions
     );
   }
 
-  let opts = toArgs(anvilOptions);
-
-  // Anvil fails to accept the `forkUrl` and `chainId` options on the Arbitrum network.
```

Comment:
did we test that this section of code is no longer needed? was it moved somewhere else? seems like not a good idea to delete